### PR TITLE
Improve language handling

### PR DESCRIPTION
**Pull request by JonaGeishauser**
Added a new helper that considers the current decimal separator when setting the separators for the tokenizer.

### CODE REVIEW

File: languages.js [hunk 1]
```diff
@@ -34,20 +34,6 @@ module.exports = {
...
-    horizontalSeparator: ',',
-    argumentSeparator: ',',
-    decimalSeparator: '.',
-    reformatNumberForJsParsing: function (n) {
-      return n;
-    },
-    isScientificNotation: function (token) {
-      return /^[1-9]{1}(\.[0-9]+)?E{1}$/.test(token);
-    }
-  },
```
Comment (JonaGeishauser) [inline languages.js:50]: Should we remove this entire languages file or keep it for now incase we have to undo our language helper logic?
Inside index.js we only use languages to display the valid language options for the tokenizer.

File: languageHelper.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  ".": {
+    verticalSeparator: ';',
+    horizontalSeparator: ',',
+    argumentSeparator: ',',
+    reformatNumberForJsParsing: reformatNumberForCommaArgumentSeparator,
+    isScientificNotation: isScientificNotationForCommaArgumentSeparator
+  }
+};
+
```
Comment (johannesboyne) [inline languageHelper.js:30]: Could be done like this as well:

```JavaScript
const SEPARATOR_MAP = {
  ",": {
    verticalSeparator: '.',
    horizontalSeparator: ';',
    argumentSeparator: ';',
    reformatNumberForJsParsing: (n) => n.replace(',', '.'),
    isScientificNotation: (token) => (/^[1-9]{1}(,[0-9]+)?E{1}$/.test(token))
  },
  ".": {
    verticalSeparator: ';',
    horizontalSeparator: ',',
    argumentSeparator: ',',
    reformatNumberForJsParsing: (n) => n,
    isScientificNotation: (token) => (/^[1-9]{1}(\.[0-9]+)?E{1}$/.test(token))
  }
};
```

**Review by johannesboyne — CHANGES_REQUESTED** [review]
WDYT, let's do this change and we're ready to go.

**Review by johannesboyne — APPROVED** [review]
LGTM

**Pull request merged**